ibm/service/vpc/resource_ibm_is_lb.go

```diff
@@ -420,9 +420,11 @@ func lbCreate(d *schema.ResourceData, meta interface{}, name, lbType, rg string,
 	}
 
 	options := &vpcv1.CreateLoadBalancerOptions{
-		IsPublic:      &isPublic,
-		IsPrivatePath: &isPrivatePath,
-		Name:          &name,
+		IsPublic: &isPublic,
+		Name:     &name,
+	}
+	if isPrivatePath {
```

Comment:
in case of false, we need not to set it ?

Reply:
Ideally API should accept false which is a default value. This seems like an API bug and i have started a conversation with the feature team. Meanwhile this fix is a workaround for terraform to not fail